pkg/packagekit/updates.jsx

```diff
@@ -1294,7 +1279,7 @@ class OsUpdates extends React.Component {
                                        if ("AllowCancel" in notify)
                                            this.setState({ allowCancel: notify.AllowCancel });
                                    })
-                .fail(ex => {
+                .catch(ex => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17517-20220705-092548-cbb7d34f-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)